python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -121,9 +122,27 @@ def __init__(self, ra=None, dec=None, source='salt2-extended'):
         self.ebvofMW = None
         if self._hascoords:
             self.mwEBVfromMaps()
+
+
+        # Behavior of model outside range :
+        # if 'zero' then all fluxes outside are set to 0.
+        self._modelOutSideRange = 'zero'
```

Comment:
You should specify that, in this case, 'range' refers to a range in time, rather than a range in wavelength.